Makefile

```diff
@@ -30,17 +28,10 @@ test-origin-conformance:
 generate-dockerfiles:
 	./openshift/ci-operator/generate-dockerfiles.sh openshift/ci-operator/knative-images $(CORE_IMAGES)
 	./openshift/ci-operator/generate-dockerfiles.sh openshift/ci-operator/knative-images in-memory-channel-controller
-	./openshift/ci-operator/generate-dockerfiles.sh openshift/ci-operator/knative-images kafka-channel-controller
-	./openshift/ci-operator/generate-dockerfiles.sh openshift/ci-operator/knative-images kafka-channel-dispatcher
 	./openshift/ci-operator/generate-dockerfiles.sh openshift/ci-operator/knative-test-images $(TEST_IMAGES)
 .PHONY: generate-dockerfiles
 
 # Generate an aggregated knative yaml file with replaced image references
 generate-release:
 	./openshift/release/generate-release.sh $(RELEASE)
-.PHONY: generate-release
-
-# Generates a ci-operator configuration for a specific branch.
-generate-ci-config:
```

Comment:
Why are you removing this? BTW, the ci-operator config needs to be re-generated now because CI failed due to the mismatch between ci-operator config and the binaries that are available on release-0.6 branch. Some images can't be built.

Reply:
OH - that is WRONG !!!!!!!!!!!!!!!!!!!!!!

Reply:
@lberk @syedriko @alanconway ^ ;-)